Makefile

```diff
@@ -339,7 +324,27 @@ clean.go: ; $(info $(H) cleaning...)
 	$(MAKE) clean -C pilot
 	$(MAKE) clean -C security
 
-test: setup go-test
+clean.installgen:
```

Comment:
Moving the section here makes it harder to correlate what generates these files and if the list is still accurate.  An alternative approach is to += the list to FILES_TO_CLEAN though I don't know how important it is for people to have higher precision on what's cleaned.

Reply:
Please add a TODO: don't generate in the source tree, they need to move to $OUT